src/components/ArticleCard.jsx

```diff
@@ -4,7 +4,9 @@ import { Link } from "react-router-dom";
 const ArticleCard = ({ article }) => {
   return (
     <section key={article.article_id} className="articlesbox">
-      <h2>{article.title}</h2>
+      <Link className="articleTitle" to={`${article.article_id}`}>
```

Comment:
Consider making more than just the title a link, it can be a bit easier and accessible if the image for example is also included in the Link element, as there's more area to click/tap ☝️